config/v1/types_ingress.go

```diff
@@ -222,3 +240,14 @@ type IngressList struct {
 
 	Items []Ingress `json:"items"`
 }
+
+// DefaultPlacement defines the default placement of ingress router pods.
+type DefaultPlacement string
+
+const (
+	// "Workers" is for having router pods placed on worker nodes by default
+	DefaultPlacementWorkers DefaultPlacement = "Workers"
+
+	// "ControlPlane" is for having router pods placed on control-plane nodes by default
+	DefaultPlacementControlPlane DefaultPlacement = "ControlPlane"
```

Comment:
Nit, can you add periods to the end of these doc strings to be consistent with other comments

Reply:
Done. Looks like these strings don't make it into the generated yaml / docs, is that to be expected? Or did I maybe mess something up here?

Reply:
I think in this case they don't end up in the generated docs you're right, but I still like to see things consistent within the code